easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -204,8 +207,13 @@ def get_toolchain_hierarchy(parent_toolchain):
                 raise EasyBuildError("No version found for subtoolchain %s in dependencies of %s",
                                      subtoolchain_name, current_tc_name)
         else:
-            raise EasyBuildError("Multiple versions of %s found in dependencies of toolchain %s: %s",
-                                 subtoolchain_name, current_tc_name, uniq_subtc_versions)
+            if subtoolchain_name == DUMMY_TOOLCHAIN_NAME:
+                # Don't care about multiple versions of dummy, take whatever comes first
+                _log.info("Ignoring multiple versions of %s in toolchain hierarchy", DUMMY_TOOLCHAIN_NAME)
+                subtoolchain_version = dep_tcs[0]['version']
```

Comment:
hmm, rather than leaving this to chance, wouldn't it be better to use `''` as version? we know that using `dummy` as version has implications (i.e. deps not being loaded at build time)